pkg/util/tracing/span_inner.go

```diff
@@ -65,6 +65,12 @@ func (s *spanInner) GetTraceRecording(recType tracingpb.RecordingType, finishing
 	return s.crdb.GetRecording(recType, finishing)
 }
 
+// GetFullTraceRecording returns the span's full recording, including detached
+// children, as a Trace. See GetTraceRecording and WithDetachedRecording.
+func (s *spanInner) GetFullTraceRecording(recType tracingpb.RecordingType) Trace {
```

Comment:
i don't understand why we don't need to check if `s.crdb != nil` but there's precedent for skipping that check.

I suppose all spans started by a `Tracer` are crdb spans.

Reply:
Yeah, this is just following the precedent of the other ones. 🤷